velox/exec/tests/PrefixSortTest.cpp

```diff
@@ -0,0 +1,329 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <gtest/gtest.h>
+#include "velox/exec/SortBuffer.h"
+
+#include "velox/exec/PrefixSort.h"
+#include "velox/exec/tests/utils/OperatorTestBase.h"
+#include "velox/exec/tests/utils/TempDirectoryPath.h"
+
+using namespace facebook::velox::exec;
+using namespace facebook::velox::exec::test;
+using namespace facebook::velox;
+using namespace facebook::velox::memory;
+using namespace facebook::velox::test;
+
+namespace facebook::velox::exec::prefixsort::test {
+
+class TestSetting {
+ public:
+  TestSetting(
+      const std::vector<CompareFlags>& sortCompareFlags,
+      const RowVectorPtr& expectedResult)
+      : sortCompareFlags_(sortCompareFlags), expectedResult_(expectedResult){};
+
+  const std::vector<CompareFlags>& sortCompareFlags() const {
+    return sortCompareFlags_;
+  }
+
+  const RowVectorPtr& expectedResult() const {
+    return expectedResult_;
+  }
+
+  std::string debugString() const {
+    const std::string expectedResultStr =
+        expectedResult_->toString(0, expectedResult_->size());
```

Comment:
If expectedResult_ has more than a handful of rows, this will be a giant string. Is this intentional?